src/chrome/content/rules/JD.com.xml

```diff
@@ -258,6 +258,7 @@
 	<target host="vc.jd.com" />
 	<target host="vcp.jd.com" />
 	<target host="vip.jd.com" />
+	<target host="vip.jr.jd.com" />
```

Comment:
Just a minor comment: Could you move this to [L202](https://github.com/jeremyn/https-everywhere/blob/fab6402ccd50b3735e8475eb0e60663dd4b09bf7/src/chrome/content/rules/JD.com.xml#L201) instead (so that the ruleset is ordered alphabetically after subdomain order)?